Bottom: 1a794c807d879f9260ba34a6d320df209f105fc8
Top:    0c3520b423bb39bc8c498ae12e4a49bbd392555d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 19:32:46 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
